### PR TITLE
Fix biopet 608

### PR DESCRIPTION
**Pull request by akaljuvee**

### CODE REVIEW

File: Common/src/main/scala/nl/lumc/sasc/biopet/test/samples/NA12878WGS.scala [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  override def configs = super.configs.::(NA12878WGS.config)
+
+}
+
+object NA12878WGS {
+
+  val config = Biopet.fixtureFile("samples", "NA12878_wgs", "Sample_U0c", "sample_config.json")
+
+  val libraries = readJsonFile(config).get("samples")
```
Comment (ffinfo) [inline Common/src/main/scala/nl/lumc/sasc/biopet/test/samples/NA12878WGS.scala]: there is a method in nl.lumc.sasc.biopet.utils.ConfigUtils that is doing this. So this means this method is duplicated.

File: Shiva/src/test/scala/nl/lumc/sasc/biopet/test/shiva/ShivaFunctional.scala [hunk 2]
```diff
@@ -78,3 +82,25 @@ class ShivaBiopetplanet30xHg19Test extends ShivaSuccess with BwaMem with Hsapien
...
+  def shouldHaveKmerContent = Some(true)
+
+  override def annotation = Some(true)
+  override def vepVersion = Some("86")
+
+  override def referenceVcf = Some(Biopet.fixtureFile("samples", "NA12878_wgs", "snp_indel_calls", "GIAB_GRCh38_v3.3.2.vcf.gz"))
+  override def referenceVcfRegions = Some(Biopet.fixtureFile("samples", "NA12878_wgs", "snp_indel_calls", "GIAB_GRCh38_regions_v3.3.2.bed"))
+
+  override def svCalling = Some(true)
+  val svCallers: List[String] = ShivaSvCalling.getSvCallersAsStrList(List(new Breakdancer, new Delly))
```
Comment (ffinfo) [inline Shiva/src/test/scala/nl/lumc/sasc/biopet/test/shiva/ShivaFunctional.scala]: Clever is working in the current develop branch. Could you add this in again?

ps: was waiting on this change to look into this PR ;)

File: Shiva/src/test/scala/nl/lumc/sasc/biopet/test/shiva/ShivaFunctional.scala [hunk 3]
```diff
@@ -78,3 +82,25 @@ class ShivaBiopetplanet30xHg19Test extends ShivaSuccess with BwaMem with Hsapien
   override def vepVersion = Some("86")
   override def memoryArg = "-Xmx1G"
 }
+
+class ShivaNA12878WGSGRCh38Test extends ShivaSuccess with BwaMem with HsapiensGRCh38_no_alt_analysis_set with NA12878WGS
+  with Haplotypecaller with HaplotypecallerGvcf with Unifiedgenotyper {
+
+  override def disablescatter = false
+  override def memoryArg = "-Xmx4G"
```
Comment (ffinfo) [inline Shiva/src/test/scala/nl/lumc/sasc/biopet/test/shiva/ShivaFunctional.scala]: Not sure with the current summary this is still needed

File: Shiva/src/test/scala/nl/lumc/sasc/biopet/test/shiva/ShivaFunctional.scala [hunk 4]
```diff
@@ -78,3 +82,25 @@ class ShivaBiopetplanet30xHg19Test extends ShivaSuccess with BwaMem with Hsapien
...
+
+  def paired = true
+  def shouldHaveKmerContent = Some(true)
+
+  override def annotation = Some(true)
+  override def vepVersion = Some("86")
+
```
Comment (ffinfo) [inline Shiva/src/test/scala/nl/lumc/sasc/biopet/test/shiva/ShivaFunctional.scala]: We should also add a dbsnp file to this run.

**Review by ffinfo — CHANGES_REQUESTED** [review]

**Comment by akaljuvee** [comment]
ok, I'm ready with these things now

**Review by rhpvorderman — APPROVED** [review]

**Review by ffinfo — APPROVED** [review]

**Pull request merged**